libimgdoc2/inc/IDocQuery3d.h

```diff
@@ -25,35 +25,35 @@ namespace imgdoc2
         /// thrown.
         ///
         /// \param          idx             The primary key of the tile.
-        /// \param [out]    coordinate           If non-null and the operation is successful, the tile-coordinate will be put here.
+        /// \param [out]    coordinate      If non-null and the operation is successful, the tile-coordinate will be put here.
         /// \param [out]    info            If non-null and the operation is successful, the logical position will be put here.
         /// \param [out]    brick_blob_info If non-null and the operation is successful, the brick-blob-info will be put here.
         virtual void ReadBrickInfo(imgdoc2::dbIndex idx, imgdoc2::ITileCoordinateMutate* coordinate, imgdoc2::LogicalPositionInfo3D* info, imgdoc2::BrickBlobInfo* brick_blob_info) = 0;
 
         /// Query the tiles table. The two query clauses are used to filter the tiles. The first clause is used to filter the tiles by their
         /// coordinates, the second by other "per tile data". The functor is called for each tile which matches the query. If the functor
-        /// returns false, the enumeration is canceled, and no more calls to the functor will occur any more. The two query clauses are
+        /// returns false, the enumeration is canceled, and no more calls to the functor will occur anymore. The two query clauses are
```

Comment:
anymore can be dropped here without losing any meaning in the sentence. Sounds better to me but only opinion based

Reply:
done, thanks!